### PR TITLE
Dog friendly

### PR DESCRIPTION
**Pull request by AdamN8142**
Adds the dog friendly boolean to dataset

### CODE REVIEW

File: src/national-parks.js [hunk 1]
```diff
@@ -440,6 +502,8 @@ const nationalParks = [
    "latitude": "20.72°N",
    "longitude": "156.17°W",
    "annualVisitors": 1112390,
+   "petsWelcome": true,
+   "editorsChoice": "Hosmer's Grove Nature Trail",
```
Comment (kaohman) [inline src/national-parks.js:506]: Add \\ before single quotes

**Review by trbachmann — APPROVED** [review]
Thanks for adding the pet and editors choice info. Didn't realize so many national parks allowed dogs, that's awesome.

File: src/national-parks.js [hunk 2]
```diff
@@ -34,6 +38,8 @@ const nationalParks = [
    "latitude": "44.60°N",
    "longitude": "110.50°W",
    "annualVisitors": 4116524,
+   "petsWelcome": true,
+   "editorsChoice": "Uncle Tom's Trail",
```
Comment (kaohman) [inline src/national-parks.js:42]: add \\ before single quote

File: src/national-parks.js (hunk 1 above)
Comment (kaohman) [reply src/national-parks.js:506]: Nevermind - since JSON only recognizes double quotes, we don't need to escape a single quote.

**Review by kaohman — APPROVED** [review]
Thanks for adding these additional details. They will be great additions to the web app.

**Pull request merged**